src/main/java/ch/njol/skript/conditions/CondIsBlockRedstonePowered.java

```diff
@@ -25,25 +25,40 @@
 import ch.njol.skript.doc.Examples;
 import ch.njol.skript.doc.Name;
 import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
 
 @Name("Is Block Redstone Powered")
-@Description("Checks if a block is powered by redstone")
+@Description("Checks if a block is (indirectly) powered by redstone")
 @Examples({"if clicked block is redstone powered:",
-	"\tsend \"This block is well-powered by redstone!\""})
+	"\tsend \"This block is well-powered by redstone!\"",
+	"if clicked block is indirectly redstone powered:",
+	"\tsend \"This block is indirectly redstone powered.\""})
 @Since("2.5")
 public class CondIsBlockRedstonePowered extends PropertyCondition<Block> {
 	
 	static {
-		register(CondIsBlockRedstonePowered.class, "redstone powered", "blocks");
+		register(CondIsBlockRedstonePowered.class, "(0¦|1¦indirectly) redstone powered", "blocks");
```

Comment:
Also, you don't need parse marks for these, you can register 2 separate condition patterns, like
```
Skript.registerCondition(CondCanHold.class,
				"%inventories% (can hold|ha(s|ve) [enough] space (for|to hold)) %itemtypes%",
				"%inventories% (can(no|')t hold|(ha(s|ve) not|ha(s|ve)n't|do[es]n't have) [enough] space (for|to hold)) %itemtypes%");
```
But you would need to change the class declaration to ``public class CondIsBlockRedstonePowered extends Condition``... although other developers might prefer parse marks, I think that's up for discussion.

Regardless, you don't need 2 parse marks for this?

Reply:
They could get rid of the 0 parse mark, making it just

(|1¦indirectly)

Reply:
or just do two patterns to reduce the needless complexity

Reply:
I'm going to use two patterns; thanks for your reviews.